cmd/lint.go

```diff
@@ -12,11 +12,16 @@ func setupLintCommand() *cobra.Command {
 		Short: "Lint the integration",
 		Long:  "Use lint command to lint the integration files.",
 		RunE:  lintCommandAction,
+		SilenceUsage: true,
 	}
 	return cmd
 }
 
 func lintCommandAction(cmd *cobra.Command, args []string) error {
-	fmt.Println("lint is not implemented yet.")
-	return nil
+	packageRootPath, err := os.Getwd()
```

Comment:
There is a method that will help you: https://github.com/elastic/elastic-package/blob/master/internal/builder/packages.go#L62 You can expose it or even extract to `internal/packages`.

Reply:
Done in 75eb0d2.